ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -664,6 +664,36 @@ FAN_SS_AL:
   - schedule_run_command
   - control_status
 
+FAN_VSC_SRC:
+  description: "Basic Exhaust fan with start/stop, speed sensor, alarms, scheduled run"
+  uses:
+  - lost_power_alarm
+  opt_uses:
+  - filter_alarm
+  implements:
+  - FAN
+  - VSC
+  - SRC
+
+FAN_VSC_SRC_DPM:
+  description: "Basic Exhaust fan with start/stop, speed sensor, fail alarm, scheduled run, dampers"
+  uses:
+  - lost_power_alarm
+  opt_uses:
+  - filter_alarm
+  implements:
```

Comment:
should also include `SS`

Reply:
Hi sir, I found VSC already has SS (run_command and run_status)

Reply:
understood but we usually include both. if you wouldnt mind adding it, it will make it a bit more consistent with the others.

Reply:
Oh I tested with DBO telemetry validation and including SS did not have error. I was expecting it errors out something like "run_command is already defined", but no it didn't. I am adding SS now sir.